lib/matplotlib/backends/backend_cairo.py

```diff
@@ -56,12 +59,6 @@
 from matplotlib.transforms   import Bbox, Affine2D
 from matplotlib.font_manager import ttfFontProperty
 
-# Image::color_conv(format) for draw_image()
```

Comment:
This came in via9cb48c163ab8f91dff03e15bb305dc238ecb1885 and it's use was removed in  #5718 and it's backports.